lib/internal/Magento/Framework/App/FeedImporterInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedImporterInterface
```

Comment:
I am not sure about this, why do you think that it should be renamed to factory, it doesn't create new objects?